content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -26,6 +26,32 @@ Thanks also to the CNCF, which provides resources and support, and to the AWS op
 
 In addition, massive thanks to Venafi for contributing developer time and resources towards the continued maintenance of cert-manager projects.
 
-## `v1.16.0`
-
-TODO
+## Changes since `v1.15.0`
+
+### Feature
+
+- Add `SecretRef` support for Venafi TPP issuer CA Bundle ([#7036](https://github.com/cert-manager/cert-manager/pull/7036), [`@sankalp-at-gh`](https://github.com/sankalp-at-gh))
```

Comment:
**nit, don't feel the need to change anything**: I wish we could write release notes that are slightly more user-focused, but I realize that it isn't that easy since each contributor writes their own release notes. Among others:

- Most of these items focus on what was technically done to achieve the feature (e.g., "add X" rather than explaining whether/how it can impact the user.
- "Features" is super heterogenous; I think the Helm improvements should be in their own section or subsection, and maybe also group "Runtime Improvements", and "API Changes".

Example:

> - Add `SecretRef` support for Venafi TPP issuer CA Bundle

should be reworded as:

> - The Venafi issuer now lets you load your custom CA bundle to connect to TPP from a Secret resource using the new field `secretRef`.

I think we would benefit greatly from having a technical writter look at our release notes from time to time. Maybe we could get @ditatechwriter (Michael) to give some advice on this.